assets/src/edit-story/components/library/libraryUploadDropTarget.js

```diff
@@ -34,16 +34,29 @@ import {
   UploadDropTargetMessageOverlay,
 } from '../uploadDropTarget';
 import { useUploader } from '../../app/uploader';
+import { useSnackbar } from '../../app/snackbar';
 
 const MESSAGE_ID = 'edit-story-library-upload-message';
 
 function LibraryUploadDropTarget({ children }) {
   const { uploadFile } = useUploader();
+  const { showSnackbar } = useSnackbar();
   const onDropHandler = useCallback(
     (files) => {
-      files.forEach(uploadFile);
+      files.forEach((file) => {
+        try {
+          uploadFile(file);
+        } catch (e) {
+          if (!e.isUserError) {
+            e.message = __('Sorry, files has failed to upload', 'web-stories');
```

Comment:
```suggestion
            e.message = __('Sorry, files have failed to upload', 'web-stories');
```

Reply:
Fixed. It was supposed to be `file has failed...` 😄

Reply:
Not fixed yet?

Reply:
Fixed!